src/actions/transactions.js

```diff
@@ -307,7 +307,30 @@ export const sent = ({
         fee: Fees.send,
         type: transactionTypes.send,
         asset: { data },
+        token: 'LSK', // TODO use the passed token once implemented
       }));
       dispatch(passphraseUsed(passphrase));
     }
   };
+
+export const updateTransactionsIfNeeded = ({ transactions, account }, windowFocus) =>
```

Comment:
this method is completely unnecessary and confusing. you can use other methods instead of this.

Reply:
I didn't create this function, I just moved it from `src/actions/account.js` when I was cleaning up that file a bit. I agree it is confusing, but I'm not sure if I can just remove it and not break anything. Either way, it's out of the scope of this pr.